src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -86,19 +87,16 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
   }
 
   protected shouldUpdate(changedProps: PropertyValues): boolean {
-    if (changedProps.has("_config")) {
+    if (hasConfigOrEntityChanged(this, changedProps)) {
```

Comment:
Right, there is an optional `entity` that is used for the `state_image` that I'm checking here. I can open a separate PR for a common function as it would hit a few other cards.